### DISCUSSION TITLE
下载课程资料时遇到证书问题

### DISCUSSION THREAD

**Discussion by flypanda999** [datawhalechina/learn-python-the-smart-way-v2#35] [提问答疑]
![aaa](https://github.com/datawhalechina/learn-python-the-smart-way-v2/assets/68719801/e3d0d134-2908-4638-a6ba-14ebfc38761f)
chapter0里的#git下载课程资料#遇到问题：在anaconda powershell prompt(mini conda 3)里输入git clone+https地址后出现如图提示，环境是win11专业版

**Comment by anine09** [comment]
Hi @flypanda999，看样子你是没安装 Steam++ 的证书，请回看 Steam++ 部分的操作视频，其中在启动后应该有一步添加证书。
